### PR TITLE
[server] adjust migration job

### PR DESCRIPTION
**Pull request by svenefftinge**
## Description

Increases the amount of users migrated per job run and adds more log information

## Related Issue(s)

Fixes #

#### Preview status

gitpod:summary

## Build Options

<details>
<summary>Build</summary>

- [ ] /werft with-werft
      Run the build with werft instead of GHA
- [ ] leeway-no-cache
- [ ] /werft no-test
      Run Leeway with `--dont-test`
</details>

<details>
<summary>Publish</summary>

- [ ] /werft publish-to-npm
- [ ] /werft publish-to-jb-marketplace
</details>

<details>
<summary>Installer</summary>

- [ ] analytics=segment
- [ ] with-dedicated-emulation
- [x] with-ws-manager-mk2
- [ ] workspace-feature-flags
  Add desired feature flags to the end of the line above, space separated
</details>

<details>
<summary>Preview Environment</summary>

- [ ] /werft with-local-preview
      If enabled this will build `install/preview`
- [ ] /werft with-preview
- [ ] /werft with-large-vm
- [ ] /werft with-gce-vm
      If enabled this will create the environment on GCE infra
- [ ] with-integration-tests=all
      Valid options are `all`, `workspace`, `webapp`, `ide`, `jetbrains`, `vscode`, `ssh`
</details>
 
/hold

### CODE REVIEW

File: components/server/src/jobs/org-only-migration-job.ts [hunk 1]
```diff
@@ -33,8 +33,9 @@ export class OrgOnlyMigrationJob implements Job {
 
             const result: User[] = [];
             for (const user of users) {
-                result.push(await this.migrationService.migrateUser(user));
+                result.push(await this.migrationService.migrateUser(user, true, this.name));
```
Comment (geropl) [inline components/server/src/jobs/org-only-migration-job.ts:36]: Before the 2nd argument was `false`, right?

Comment (svenefftinge) [reply components/server/src/jobs/org-only-migration-job.ts:36]: yes, but it should be true

**Review by geropl — APPROVED** [review]
LGTM! :heavy_check_mark:

**Pull request merged**